src/07-even-more-loops.js

```diff
@@ -57,7 +81,17 @@ function shoutForOfLoop(array) {}
  *  sumArray([ 10, 0, 10, 11 ]);
  *  //> 31
  */
-function sumArray(array) {}
+function sumArray(array) {
+let sum = 0;
+let newArr = [];
+
+for (let i = 0; i < array.length; i++) {
+   newArr.push(array[i])
+sum = newArr.reduce((a, b) => a + b);
```

Comment:
In this function, if you use reduce you shouldn't use a loop. Since reduce is lopping through the array, you can apply the method directly tot the original `array` and it will give you the same outcome. 

Have reduce inside the loop, is almost like a nested loop and is running on every iteration.